src/Catalog/Persistence/Storage.cs

```diff
@@ -170,6 +170,17 @@ public Uri ResolveUri(string relativeUri)
             return new Uri(BaseAddress, relativeUri);
         }
 
+        /// <summary>
+        /// It will return true if there are changes between the source and destination.
+        /// </summary>
+        /// <param name="sourceUri">The soource uri.</param>
+        /// <param name="destinationUri">The destination uri.</param>
+        /// <returns></returns>
+        public virtual Task<bool> AreSyncronized(Uri sourceUri, Uri destinationUri)
+        {
+            return Task.FromResult(true);
```

Comment:
This should be false by default. Having this as true breaks Catalog2Dnx with `FileStorage` because it will always think that it doesn't need to save the file because it's synchronized.